kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2035,15 +2035,75 @@ object KyuubiConf {
         "and groups information for different user or session configs. This config value " +
         "should be a class which is a child of 'org.apache.kyuubi.plugin.GroupProvider' which " +
         "has zero-arg constructor. Kyuubi provides the following built-in implementations: " +
-        "<li>hadoop: delegate the user group mapping to hadoop UserGroupInformation.</li>")
+        "<li>hadoop: delegate the user group mapping to hadoop UserGroupInformation.</li> " +
+        "<li>ldap: delegate the user group mapping to ldap.</li>")
       .version("1.7.0")
       .stringConf
       .transform {
         case "hadoop" => "org.apache.kyuubi.session.HadoopGroupProvider"
+        case "ldap" => "org.apache.kyuubi.session.LDAPGroupProvider"
         case other => other
       }
       .createWithDefault("hadoop")
 
+  val LDAP_GROUP_PROVIDER_URL: OptionalConfigEntry[String] =
+    buildConf("kyuubi.session.group.ldap.url")
+      .doc("SPACE character separated LDAP connection URL(s).")
+      .version("1.0.0")
```

Comment:
the version is not correct, it should be `1.7.0`

Reply:
thanks,i will fix it